docs: add no proxy list configuration for VMO PEM-5233 (#3586)
spectrocloud/librarium: docs/docs-content/vm-management/install-vmo-in-airgap.md

  [Self-Hosted Palette Installation](../enterprise-version/install-palette/install-palette.md) and
  [Palette VerteX Installation](../vertex/install-palette-vertex/install-palette-vertex.md) guides for more information.

  :::info

  If your environment is configured to use an external proxy, you can use the **No Proxy** list to exclude any domains

>   or IP addresses from proxying. You have the option to set the No Proxy list during the Palette installation flow. This
>   is useful for scenarios where you know the IP addresses of your VMs load balancers before deployment.

Reviewer: ```suggestion
  or IP addresses from proxying. You have the option to set the No Proxy list during the Palette installation flow. This is useful in scenarios where you know the IP addresses you want to exclude before deployment, such as load balancer IP addresses.
```
maybe??
